### DISCUSSION TITLE
Why is my Mastodon account locked for no reason?

### DISCUSSION THREAD

**Discussion by HoaiThuDao** [mastodon/mastodon#19270] [Q&A]
My Mastodon account is locked even if I don't post offending content or make comments that violate Mastodon's rules. The content of the Mastodon account lock notice is as follows:

**"You can no longer use your account, your profile and other data are no longer accessible. For 30 days, you can still sign in to request a copy of your data until it's completely deleted, but we'll retain some basic data to prevent you from getting out of invalidation. "**

Can anyone help me because I can't find the information of Mastodon customer service. Thank you very much!

**Comment by krisfail** [comment]
This is the place to talk about the system that runs Mastodon. Questions about individual servers will probably not be answered by anyone here.
